examples/official-storybook/preview.js

```diff
@@ -216,7 +216,14 @@ export const globalTypes = {
     defaultValue: 'en',
     toolbar: {
       icon: 'globe',
+      shortcuts: {
```

Comment:
I tried adding a `previous` shortcut but it appears to do the same thing as `next`. Can you double check this?

Reply:
Will look into this - will still just cycle, but backwards

Reply:
Works now - example uses L to cycle forwards between locales and K to cycle backwards (and it won't hit the "reset" menu item)

Reply:
Looking good